ext/openssl/tests/openssl_get_cert_locations_basic.phpt

```diff
@@ -0,0 +1,11 @@
+--TEST--
+openssl_get_cert_locations() tests
+--SKIPIF--
+<?php if (!extension_loaded("openssl")) print "skip"; ?>
+--FILE--
+<?php
+// openssl locations differ per distro.
+var_dump(count(openssl_get_cert_locations()));
+?>
+--EXPECTF--
+int(8)
```

Comment:
It looks like [the function source code](https://lxr.room11.org/xref/php-src%40master/ext/openssl/openssl.c#1626) will always create an 8-element array. Given this, the test is pretty pointless.